src/components/mixins/inputValidation.js

```diff
@@ -19,8 +19,7 @@ const set = {
     { pattern: /^([a-z0-9\-.]*)\.(([a-z]{2,4})|([0-9]{1,3}\.([0-9]{1,3})\.([0-9]{1,3})))|(:[0-9]{1,5})$/, message: 'Invalid IP', trigger: 'change' }
   ],
   walletAddress: [
-    { required: true, message: 'Please input wallet address', trigger: 'change' },
-    { pattern: /^0x[a-fA-F0-9]{40}$/, message: 'Invalid wallet address', trigger: 'change' }
```

Comment:
I'm not sure but doesn't it work if add both BTC and ETH patterns here? 🤔

Reply:
I'm trying to do something like this:
``` javascript
  walletAddress: [
    { required: true, message: 'Please input wallet address', trigger: 'change' },
    { pattern: /^[13][a-km-zA-HJ-NP-Z1-9]{25,34}$/, message: 'Invalid wallet address', trigger: 'change' },
    { pattern: /^0x[a-fA-F0-9]{40}$/, message: 'Invalid wallet address', trigger: 'change' }
  ]
```
but this don't work. It always says that wallet address is invalid.